src/domloader.ml

```diff
@@ -1,5 +1,7 @@
-(* just here to allow other modules to load the raw code below *)
-let init () = ()
+(** Initialize the Node environment to make it ready to run tests *)
+let init () =
+  (* this function doesn't do anything, it just allows other modules to load the raw code below *)
+  ()
 
 [%%mel.raw
 {|
```

Comment:
@jchavarri This JS code below, was it copied from somewhere? If so, maybe we should add a link to the source?

Reply:
Added link in https://github.com/ahrefs/melange-fest/pull/7/commits/cc56e26c55a9376aab4b7fb2f534f7567cffd011